browser-interface/packages/shared/chat/sagas.ts

```diff
@@ -296,7 +296,7 @@ function initChatCommands() {
       .filter(([_, value]) => value.position)
       .map(function ([, value]) {
         // eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
-        const name = getProfile(store.getState(), value.ethereumAddress!)?.name ?? 'unknown'
+        const name = profile(store.getState(), value.ethereumAddress!)?.name ?? 'unknown'
```

Comment:
I think the `getProfile` is better in this context. It is hard to tell whether the `profile` function has side effects or what it does by the name. Does it create, gets or fetches a profile?